.gitignore

```diff
@@ -1,9 +1,3 @@
-target/
-# IntelliJ local files
-*.idea
-*.iml
-nbactions.xml
-maven-deps/
-.DS_Store
-archived/
-cygnus-ngsi/ignore
+.DS_Store	.DS_Store
+archived/	archived/
+cygnus-ngsi/ignore
```

Comment:
I think we should keep previous version of this file

Reply:
Done